File: kamino/src/main/java/org/example/product/kamino/KaminoModule.java
package org.example.product.kamino;

import org.example.product.coruscant.CoruscantModuleData;
import org.jboss.resteasy.plugins.guice.GuiceResteasyBootstrapServletContextListener;
import org.jboss.resteasy.plugins.providers.jackson.ResteasyJackson2Provider;
import org.jboss.resteasy.plugins.server.servlet.ResteasyBootstrap;

/**
 * I am published and therefore I have Javadoc!
 */
public class KaminoModule {

    /**
     * Bootstrap...
     */
    public ResteasyBootstrap bootstrap;

    /**
     * Data...
     */
    public CoruscantModuleData data;

    /**
     * Info!
     *
     * @return all the important Classes
     */
    public Class<?>[] info() {
        return new Class<?>[] {
            ResteasyBootstrap.class, ResteasyJackson2Provider.class, GuiceResteasyBootstrapServletContextListener.class
        };
    }
}

File: kamino/src/main/kotlin/org/example/product/kamino/KaminoModule.kt
package org.example.product.kamino

import org.example.product.coruscant.CoruscantModuleData

/** I am published and therefore I have Javadoc! */
class KaminoModule {

    /** Data... */
    var data: CoruscantModuleData? = null

    /**
     * Info!
     *
     * @return all the important Classes
     */
    fun info(): Array<Class<*>> {
        return arrayOf()
    }
}
